package.json

```diff
@@ -8,7 +8,7 @@
     "test": "ng test",
     "lint": "ng lint",
     "commit": "git-cz",
-    "build:ci": "ng build --prod hyperdash-angular && ng build --prod hyperdash-angular-app",
+    "build:ci": "ng build --configuration production hyperdash-angular && ng build --configuration production hyperdash-angular-app",
```

Comment:
do we need to add a configuration flag here? I think Angular moving towards prod build by default. I would say let opt in for this feature. 

@aaron-steinfeld thoughts?

```
You can run the optional migration for enabling production builds by default ng update @angular/cli@12 --migrate-only production-by-default.

```

Reply:
We should also enable `NG_PERSISTENT_BUILD_CACHE` for faster build.

https://github.com/angular/angular-cli/pull/20756

```
  "start": "NG_PERSISTENT_BUILD_CACHE=1 ng serve",
  "build": "NG_PERSISTENT_BUILD_CACHE=1 ng build",
```

Reply:
> I would say let's opt in for this feature.

So we can certainly compare build times, but does this mean our local builds would be minified, not source mapped etc? If so, I'd be against it. In other words, what is the advantage of doing a prod build locally other than a greater fidelity to the final state?

Reply:
They have better template checking. I would also not want to miss out on source map.